test-framework/clustering/src/main/java/org/keycloak/testframework/server/ClusteredKeycloakServer.java

```diff
@@ -94,12 +111,16 @@ private void startContainersWithSameImage(KeycloakServerConfigBuilder configBuil
             containers[i] = container;
 
             copyProvidersAndConfigs(container, configBuilder);
-
-            container.setCustomLogConsumer(new JBossLogConsumer(Logger.getLogger("managed.keycloak." + i)));
+            configureLogConsumers(container, i, clusterLatch);
             container.run(configBuilder.toArgs());
         }
     }
 
+    private void configureLogConsumers(DockerKeycloakDistribution container, int index, CountdownLatchLoggingConsumer clusterLatch) {
```

Comment:
nitpick, can be static
```suggestion
    private static void configureLogConsumers(DockerKeycloakDistribution container, int index, CountdownLatchLoggingConsumer clusterLatch) {
```